src/lifecycle/properties-init.js

```diff
@@ -79,13 +79,20 @@ function createNativePropertyDefinition (name, opts) {
 
   prop.get = function () {
     const info = data(this, `api/property/${name}`);
+    const internalValue = info.internalValue;
 
     if (opts.get) {
-      return opts.get(this);
+      return opts.get(this, { name, internalValue });
```

Comment:
Is this new API? i.e. you have access to an object with `internalValue` when you call the `get()` method?